pymc4/distributions/distribution.py

```diff
@@ -202,6 +203,10 @@ class BoundedDiscreteDistribution(DiscreteDistribution, BoundedDistribution):
     ...
 
 
+class BoundedDiscreteCompoundDistribution(BoundedDiscreteDistribution):
```

Comment:
Could you explain why we should add the `BoundedDiscreteCompoundDistribution` distribution and not use `BoundedDiscreteDistribution` directly?

Reply:
If we add other compound distributions in the future than this would be a more interpretable parent class. But it's not necessary! If you want, I can remove it...

Reply:
My question was more about, what make a _compound distribution_ different from a _distribution_ from a programmers point of view. I understand that they are different mathematical entities, but what does the compoundness change from the `class` inheritance point of view.

Reply:
Hmm... You are right! It will mostly not affect anything during coding. I will change that too! Thanks